src/current/v23.2/molt-fetch.md

```diff
@@ -0,0 +1,438 @@
+
+### Data path
+
+MOLT Fetch can move the source data to CockroachDB via [cloud storage](#cloud-storage), a [local file server](#local-file-server), or [directly](#direct-copy) without an intermediate store.
+
+#### Cloud storage
+
+`--bucket-path` specifies that MOLT Fetch should write intermediate files to a path within a [Google Cloud Storage](https://cloud.google.com/storage/docs/buckets) or [Amazon S3](https://aws.amazon.com/s3/) bucket to which you have the necessary permissions. For example:
+
+Google Cloud Storage:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--bucket-path 'gs://migration/data/cockroach'
+~~~
+
+Amazon S3:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--bucket-path 's3://migration/data/cockroach'
+~~~
+
+Cloud storage can be used with either the [`IMPORT INTO` or `COPY FROM` modes](#fetch-mode).
+
+#### Local file server
+
+`--local-path` specifies that MOLT Fetch should write intermediate files to a path within a [local file server]({% link {{ page.version.version }}/use-a-local-file-server.md %}). `local-path-listen-addr` specifies the address of the local file server. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--local-path /migration/data/cockroach
+--local-path-listen-addr 'localhost:3000'
+~~~
+
+In some cases, CockroachDB will not be able to use the local address specified by `--local-path-listen-addr`. This can depend on where CockroachDB is deployed, the runtime OS, and the source dialect.
+
+For example, if you are migrating to CockroachDB {{ site.data.products.cloud }}, such that the {{ site.data.products.cloud }} cluster is in a different physical location than the machine running `molt fetch`, then CockroachDB cannot reach an address such as `localhost:3000`. In these situations, use `--local-path-crdb-access-addr` to specify an address for the local file server that is reachable by CockroachDB. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--local-path /migration/data/cockroach
+--local-path-listen-addr 'localhost:3000'
+--local-path-crdb-access-addr '44.55.66.77:3000'
+~~~
+
+A local file server can be used with either the [`IMPORT INTO` or `COPY FROM` modes](#fetch-mode).
+
+{{site.data.alerts.callout_success}}
+[Cloud storage](#cloud-storage) is often preferable to a local file server, which can require considerable disk space.
+{{site.data.alerts.end}}
+
+#### Direct copy
+
+`--direct-copy` specifies that MOLT Fetch should use `COPY FROM` to move the source data directly to CockroachDB without an intermediate store:
+
+- Because the data is held in memory, the machine must have sufficient RAM for the amount of data being moved.
+- Direct copy mode does not support compression or [continuation](#fetch-continuation).
+- The [`--use-copy`](#fetch-mode) flag is redundant with `--direct-copy`.
+
+### Schema and table selection
+
+By default, MOLT Fetch moves all source data to CockroachDB. Use the following flags to move a subset of data.
```

Comment:
```suggestion
By default, MOLT Fetch moves all data to CockroachDB that is contained in the database specified in the source connection string. Use the following flags to move a subset of data.
```

I'm not in love with the proposed wording here, so feel free to reject or massage further.

Reply:
This may be simplest: ```By default, MOLT Fetch moves all data from the [`--source`](#source-and-target-databases) database to CockroachDB.```